eoscostarica/lifebank
# Add landing page breakpoint

## Add Desktop breakpoint version

### What does this PR do?
Add desktop breakpoint for landing page

### How should this be tested?
- make run
- go to `http://localhost:3000`
- see landing page
